src/main/java/org/catrobat/jira/timesheet/activeobjects/impl/TimesheetImpl.java

```diff
@@ -41,4 +41,23 @@ public TimesheetEntry firstEntry() {
 
         return entries.get(0);
     }
+
+    public final int calculateTotalHours()
```

Comment:
Please use uniform coding style (no new line for curly brackets).

Reply:
corrected brackets